main.py

```diff
@@ -0,0 +1,100 @@
+""" Shest Api Server """
```

Comment:
```suggestion
""" SHEST (Shell Over REST) Api Server """
```
Looks better in my opinion.